applications/launchpad/backend/src/main.rs

```diff
@@ -47,9 +47,18 @@ use crate::{
     grpc::GrpcWalletClient,
 };
 
+use tauri_plugin_sql::{Migration, MigrationKind, TauriSql};
+
 #[tokio::main]
-async fn main() {
+async fn spawn_grpc() {
+    tokio::spawn(subscribe());
+}
+
+fn main() {
     env_logger::init();
+
+    spawn_grpc();
```

Comment:
@CjS77 @alex1307 
is this acceptable?
I had to do this, because tauri-sql-plugin in its `initialize` function calls `block_on`, which tokio panics about, because with async main it conflicted with the thread being used to manage async operations

this is the only way I found to run both grpc and the tauri-sql-plugin
tried `tauri::async_runtime::set(tokio::runtime::Handle::current());`, but it still complained that you "cannot start runtime within runtime"

Reply:
#worksOnMyMachine

have you tried running `yarn` in both /launchpad and /launchpad/gui-react ?
yarn.lock has the newest dependencies that should work (and if you update only one of them they might tauri cli might be incompatible with actual tauri package in react)

Reply:
Solved. We had to upgrade cargo tauri.

If you were using `yarn run tauri ...` you would not have had this issue.

So `cargo install tauri-cli --version 1.0.0-rc.13` does the trick.

Other things that can help:
1. `cargo clean`
1. `cargo cache -a`